hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -427,4 +419,9 @@ public void postMasterStartupInitialize() {
   public void updateBalancerStatus(boolean status) {
     internalBalancer.updateBalancerStatus(status);
   }
+
+  @Override
+  public List<RegionPlan> balanceTable(TableName tableName, Map<ServerName, List<RegionInfo>> clusterLoad){
```

Comment:
Why not support here?

Reply:
yes , I think 'balanceTable' should not be method of LoadBalancer , it should be method of BaseLoadBalancer , and RSGroupBasedLoadBalancer no need implement it